app/assets/stylesheets/common/base/discourse.scss

```diff
@@ -297,7 +297,6 @@ table {
   .d-icon {
     margin-left: 0.25em;
   }
-  @include unselectable;
```

Comment:
I think we can still keep the `unselectable` mixin, but maybe `cursor: pointer` should be moved after it, so it overrides `cursor: default`?